### ISSUE TITLE
Import error in gluoncv segbase.py

### ISSUE THREAD

**Issue by ThomasSELECK** [dmlc/gluon-cv#887]
Hello everyone,

I try to create a pretrained FCN model using GluonCV. But it doesn't work properly.

Minimal reproducible example:

```
import mxnet as mx
import gluoncv
model = gluoncv.model_zoo.get_fcn(dataset = 'pascal_voc', backbone='resnet34', pretrained = True, ctx = mx.gpu(0))
```

Error returned:
```
Traceback (most recent call last):
  File "D:\test.py", line 3, in <module>
    model = gluoncv.model_zoo.get_fcn(dataset = 'pascal_voc', backbone='resnet34', pretrained = True, ctx = mx.gpu(0))
  File "C:\Program Files\Python37\lib\site-packages\gluoncv\model_zoo\fcn.py", line 126, in get_fcn
    ctx=ctx, **kwargs)
  File "C:\Program Files\Python37\lib\site-packages\gluoncv\model_zoo\fcn.py", line 46, in __init__
    crop_size=crop_size, pretrained_base=pretrained_base, **kwargs)
  File "C:\Program Files\Python37\lib\site-packages\gluoncv\model_zoo\segbase.py", line 49, in __init__
    pretrained = resnet34_v1b(pretrained=pretrained_base, dilated=True, **kwargs)
NameError: name 'resnet34_v1b' is not defined
```

Updating the imports in the file "C:\Program Files\Python37\lib\site-packages\gluoncv\model_zoo\segbase.py" at line 8, replacing
```
from .resnetv1b import resnet50_v1s, resnet101_v1s, resnet152_v1s
```
by
```
from .resnetv1b import resnet18_v1b, resnet34_v1b, resnet50_v1s, resnet101_v1s, resnet152_v1s
```

Leads to a new error:
```
Traceback (most recent call last):
  File "D:\test.py", line 3, in <module>
    model = gluoncv.model_zoo.get_fcn(dataset = 'pascal_voc', backbone='resnet34', pretrained = True, ctx = mx.gpu(0))
  File "C:\Program Files\Python37\lib\site-packages\gluoncv\model_zoo\fcn.py", line 130, in get_fcn
    'fcn_%s_%s'%(backbone, acronyms[dataset]), tag=pretrained, root=root), ctx=ctx)
  File "C:\Program Files\Python37\lib\site-packages\gluoncv\model_zoo\model_store.py", line 173, in get_model_file
    short_hash=short_hash(name))
  File "C:\Program Files\Python37\lib\site-packages\gluoncv\model_zoo\model_store.py", line 142, in short_hash
    raise ValueError('Pretrained model for {name} is not available.'.format(name=name))
ValueError: Pretrained model for fcn_resnet34_voc is not available.
```

Note: This error also come when the `backbone` argument of the method `gluoncv.model_zoo.get_fcn` is "resnet18".

Environment:
Windows 10 x64
CUDA 10.0
Python 3.7.2
MxNet 1.4.1
GluonCV 0.5.0b20190704

**Comment by YimianDai** [comment]
It seems that only `fcn_resnet50_voc`, `fcn_resnet101_voc`, `fcn_resnet101_coco`, `fcn_resnet50_ade` and `fcn_resnet101_ade` have the pretrained weights.

**Comment by ThomasSELECK** [comment]
I also tried with `fcn_resnet50_voc`, but weights are not available too.

**Comment by zhanghang1989** [comment]
We have unitests covered all the pretrained models. I believe you may have some internet issue.

https://github.com/dmlc/gluon-cv/blob/master/tests/unittests/test_model_zoo.py#L349-L356

**Comment by wangx404** [comment]
> We have unitests showing the covered pretrained models.
> https://github.com/dmlc/gluon-cv/blob/master/tests/unittests/test_model_zoo.py#L349-L356
> 
> Some of the pretrained models are not provided. You may train it on you machine.

The problem is actually you have forgotten importing `resnet18_v1b` in `model_zoo/segbase.py`. From the importing part in this python script which says `from .resnetv1b import resnet50_v1s, resnet101_v1s, resnet152_v1s`, we can see `resnet18_v1b` is missing. This is the real reason for import error. Hope it can be fixed as soon as possible.

**Comment by bryanyzhu** [comment]
The import error has been fixed in PR #1155 , thank you for reporting it.
